lib/order.rb

```diff
@@ -0,0 +1,29 @@
+require_relative 'menu'
+require_relative 'takeaway'
+
+class Order
+  attr_reader :basket, :menu
+
+  def initialize
+    @basket = {}
+    @basket_total = 0
+    @menu = Menu.new
+  end
+
+  def add_to_order(dish, quantity)
+    @basket[dish] = quantity
+  end
+
+  def order_summary
+    @basket.each do |item, quantity|
+      puts "#{item} x #{quantity}: £#{Menu.new.menu[item] * quantity}"
```

Comment:
rather than relying on the `Menu` class directly here, could we use `@menu`? Combined with dependency injection this would allow us to isolate the two classes